models/flag.js

```diff
@@ -0,0 +1,21 @@
+const firestore = require("../utils/firestore");
+const flagModel = firestore.collection("featureFlags");
+
+/**
+ *
+ * @param flagData { Object }: flag data object to be stored in DB
+ * @returns Flag Id: String
+ */
```

Comment:
Please add a description for `addFlag` function

Reply:
done